code-challenges.test.js

```diff
@@ -48,6 +79,31 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+ 
+describe("nouns", () => {
+    it("takes in an array of words and returns an array with all the words capitalized")
```

Comment:
Don't forget the it block will also need an anonymous function passed into it.  Feel free to copy the code from the syllabus and modify it to fit the needs of the particular problem.